### PR TITLE
Fix #48

### LINKED ISSUE #48: PWA (manifest) logo icons are not being loaded
**Describe the bug**

* PWA logo icons are not present

**Expected behavior**

* Logo images should be present and hosted

**To Reproduce**

* Go to https://react-native-elements.js.org/#/
* See console
image
* The required app logo images are not present

### PR DESCRIPTION
**Pull request by iamtariqul**
#48 
Screenshot from 2021-03-19 19-30-11

Done all this problem.

### CODE REVIEW

**Comment by Uyadav207** [comment]
@tstariqul, Fix already made.
Duplicate of #49

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }	    
-  ],
+  "icons": [{
```
Comment (pranshuchittora) [inline public/manifest.json:4]: 3 variants of icons pushed via a merged PR

File: src/components/Footer/Footer.module.css [hunk 2]
```diff
@@ -11,11 +11,12 @@ footer {
   font-weight: 200;
   font-style: italic;
   color: #cac8c8;
+  position: absolute;
 }
 
 .footer_product_img {
   width: 150px;
-  padding: 1rem 0;
+  padding: 2rem 0 0.5rem 0;
```
Comment (pranshuchittora) [inline src/components/Footer/Footer.module.css:19]: ??

File: src/components/Footer/index.jsx [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
-  Grid,
-  ListItem,
-  Typography,
-  Button,
-  Divider,
+  Divider, Grid,
+
+  Typography
 } from "@material-ui/core";
+import React from "react";
```
Comment (pranshuchittora) [inline src/components/Footer/index.jsx:6]: React must be on line 1

**Review by pranshuchittora — CHANGES_REQUESTED** [review]
Left some comments. Pls check

**Pull request closed without merging**